core/src/main/java/org/apache/carbondata/core/constants/CarbonCommonConstants.java

```diff
@@ -1845,6 +1845,21 @@
   public static final int CARBON_MINMAX_ALLOWED_BYTE_COUNT_MIN = 10;
   public static final int CARBON_MINMAX_ALLOWED_BYTE_COUNT_MAX = 1000;
 
+  /**
+   * Written by detail to be written in carbondata footer for better maintanability
+   */
+  public static final String CARBON_WRITTEN_BY_FOOTER_INFO = "written_by";
+
+  /**
+   * carbon version detail to be written in carbondata footer for better maintanability
+   */
+  public static final String CARBON_VERSION_FOOTER_INFO = "version";
```

Comment:
I think it is better to give a more specific name instead of "version". How about "carbon_writter_version"
And in the comment, you can say it is CarbonData software version used when writing the carbon files

Reply:
yes, even this suits better